patitasuy/src/app/components/post-card/post-card.component.html

```diff
@@ -1,7 +1,8 @@
-<mat-card routerLink="/details/{{post.id}}">
-  <img mat-card-image src="{{post.url}}" alt="Pet image">
+<mat-card class="custom-card" routerLink="/details/{{post.id}}">
+  <img class="image" src="{{post.url}}" alt="Pet image">
   <mat-card-content>
     <p>{{post.title}}</p>
     <p>{{post.location}}</p>
+    <p class="state">{{post.state}}</p>
```

Comment:
Make sure the status is only being displayed on the profile view, home view should not display the status